src/main/resources/static/js/login.js

```diff
@@ -1,28 +1,13 @@
 jQuery(document).ready(function () {
+    function reroutToSubscritpion() {
+        checkBackendSecured();
+        // if (ldapEnabled == false || (ldapEnabled == true && currentUser != null)) {
+        if (isLdapEnabled() && currentUser != null && currentUser != undefined || !isLdapEnabled()) {
```

Comment:
Why is last "!isLdapEnabled()" needed? It should be first in the if statement,, no need of doing the other checks before if isLdapEnabled is disabled.

Reply:
Done. It should only be !isLdapEnabled() cause if ldap is disabled then we should not show login page, and if ldap is not disabled then it must of course be true, så the other check for ldap is not needed.